### PR TITLE
feature/SM-6707-View S74 details

### PR DESCRIPTION
**Pull request by RichardEustace**

### CODE REVIEW

File: docs/_includes/api-documentation/v3/access-and-permissions.md [hunk 1]
```diff
@@ -68,16 +68,13 @@ The Reporting and Data Export APIs will automatically filter data in endpoint re
 `HighwayAuthority` users are not restricted to work records based upon workstreams or contract associations. Instead they are only allowed to perform write actions on resources associated with their own organisation. For example they can only assess permits for work records which have been assigned to their own organisation. `HighwayAuthority` users can instead filter resources using Geographical Areas. For more information see the [resource guide](#geographical-areas)
 {: .govuk-body}
 
-#### Street Works Administrators
+#### Role Based Access - StreetWorksAdmin
 {: .govuk-heading-s}
 
-The `Street Works Administrator` role cannot be used by itself, it must be paired with either Admin, Planner or Highway Authority. This role is not available to contractors.
+The `StreetWorksAdmin` role cannot be used by itself, it must be paired with either Admin, Planner or Highway Authority. This role is not available to contractors. Users who are have been granted the `StreetWorksAdmin` role are able to access additional sensitive information, for example financial information relating to Section 75s.
```
Comment (ccurran01) [inline docs/_includes/api-documentation/v3/access-and-permissions.md:74]:
```suggestion
The `StreetWorksAdmin` role cannot be used by itself, it must be paired with either Admin, Planner or Highway Authority. This role is not available to contractors. Users who are have been granted the `StreetWorksAdmin` role are able to access additional sensitive information, for example financial information relating to Section 74s.
```

File: docs/_includes/api-documentation/v3/resource-guide.md [hunk 2]
```diff
@@ -845,9 +845,22 @@ The POST endpoint will start a job which will generate sample inspections.
...
 
+The GET endpoint will require the section 74 reference number of the intended section 74 to return information. If the user has the `StreetWorksAdmin` role then additional sensitive financial information will be returned.  The additional properties are -
+
+<ol class="govuk-list govuk-list--bullet">
+  <li>draft_invoice_reference</li>
+  <li>draft_invoice_amount</li>
+  <li>invoice_reference (returned if <code>ha_status_id = `RefSection74HAStatus.resolved</code></li>
+  <li>final_agreed_amount (returned if <code>ha_status_id = `RefSection74HAStatus.resolved</code></li>
+</ol>
+{: .govuk-body}
```
Comment (ccurran01) [inline docs/_includes/api-documentation/v3/resource-guide.md:861]: I think this class tag might need moved above this ol?

File: docs/_includes/api-documentation/v3/resource-guide.md [hunk 3]
```diff
@@ -845,9 +845,21 @@ The POST endpoint will start a job which will generate sample inspections.
...
+
+The GET endpoint will require the section 74 reference number of the intended section 74 to return information. If the user has the `StreetWorksAdmin` role then additional sensitive financial information will be returned.  The additional properties are -
 {: .govuk-body}
 
+<ol class="govuk-list govuk-list--bullet">
+  <li>draft_invoice_reference</li>
+  <li>draft_invoice_amount</li>
+  <li>invoice_reference (returned if <code>ha_status_id = `RefSection74HAStatus.resolved</code></li>
+  <li>final_agreed_amount (returned if <code>ha_status_id = `RefSection74HAStatus.resolved</code></li>
```
Comment (ccurran01) [inline docs/_includes/api-documentation/v3/resource-guide.md:860]:
```suggestion
  <li>invoice_reference (returned if <code>ha_status_id = RefSection74HAStatus.resolved</code></li>
  <li>final_agreed_amount (returned if <code>ha_status_id = RefSection74HAStatus.resolved</code></li>
```

File: docs/_includes/api-documentation/v3/access-and-permissions.md [hunk 4]
```diff
@@ -68,16 +68,13 @@ The Reporting and Data Export APIs will automatically filter data in endpoint re
...
 
-The `Street Works Administrator` role cannot be used by itself, it must be paired with either Admin, Planner or Highway Authority. This role is not available to contractors.
+The `StreetWorksAdmin` role cannot be used by itself, it must be paired with either Admin, Planner or Highway Authority. This role is not available to contractors. Users who are have been granted the `StreetWorksAdmin` role are able to access additional sensitive information, for example financial information relating to Section 74s.
 {: .govuk-body}
 
-In the future the `Street Works Administrator` role will be used to give some trusted users elevated access to sensitive information, for example financial information. This role has no effect at this time.
-{: .govuk-body}
-
-The `Street Works Administrator` role is not the same as the `User Administrator` role and does not provide access to functionality that is currently exclusive to `User Administrator`s.
+The `StreetWorksAdmin` role is not the same as the `User Administrator` role and does not provide access to functionality that is currently exclusive to `User Administrator`s.
```
Comment (caoimheb) [inline docs/_includes/api-documentation/v3/access-and-permissions.md:77]:
```suggestion
The `StreetWorksAdmin` role is not the same as the `Admin` role and does not provide access to functionality that is currently exclusive to `Admin`s.
```
If we are using the names in the list at the top of the "Role Based Access" section

File: docs/_includes/api-documentation/v3/resource-guide.md [hunk 5]
```diff
@@ -845,9 +845,21 @@ The POST endpoint will start a job which will generate sample inspections.
...
+The POST endpoint will create a section 74 and return a section 74 reference number.
+{: .govuk-body}
+
+The GET endpoint will require the section 74 reference number of the intended section 74 to return information. If the user has the `StreetWorksAdmin` role then additional sensitive financial information will be returned.  The additional properties are -
 {: .govuk-body}
 
+<ol class="govuk-list govuk-list--bullet">
+  <li>draft_invoice_reference</li>
+  <li>draft_invoice_amount</li>
+  <li>invoice_reference (returned if <code>ha_status_id = RefSection74HAStatus.resolved</code></li>
```
Comment (cpaul92) [inline docs/_includes/api-documentation/v3/resource-guide.md:859]: `RefSection74HAStatus` is an internal type, this should be `Section74HAStatusResponse` instead

File: docs/_includes/api-documentation/v3/resource-guide.md [hunk 6]
```diff
@@ -845,9 +845,21 @@ The POST endpoint will start a job which will generate sample inspections.
...
+  <li>final_agreed_amount (returned if <code>ha_status_id = RefSection74HAStatus.resolved</code></li>
```
Comment (cpaul92) [inline docs/_includes/api-documentation/v3/resource-guide.md:860]: same here

**Pull request merged**